code/game/gamemodes/changeling/implements/powers/suck.dm

```diff
@@ -10,6 +10,10 @@
 	if(!istype(G))
 		to_chat(src, "<span class='warning'>We must be grabbing a creature in our active hand to absorb them.</span>")
 		return
+	var/mob/living/carbon/human/ling = src
+	if(istype(ling) && !ling.get_pressure_weakness())
```

Comment:
The first check is useless, because src is basically never null during this. So you can safely remove the typecast and istype(ling).

Reply:
src is only type mob; I think it fails to compile without an explicit cast

Reply:
Yea it does fail to compile without an explicit var cast to type human; the check might still be kinda redundant though so I can take that out if you'd like @NonQueueingMatt ?

Reply:
Casting src is something that shouldn't happen in these cases. Remove the cast and make `get_pressure_weakness` a /mob/ level proc that's overridden at human level.

Reply:
Okay! :)